services/user.py

```diff
@@ -0,0 +1,45 @@
+from typing import Optional
+
+from django.contrib.auth import get_user_model
+from django.db.models import QuerySet
+
+from settings import AUTH_USER_MODEL
+
+
+def create_user(username: str,
+                password: str,
+                email: Optional[str] = "",
+                first_name: Optional[str] = "",
+                last_name: Optional[str] = "") -> AUTH_USER_MODEL:
+    creating_user = get_user_model().objects.create_user(
+        username=username,
+        password=password,
+        email=email,
+        first_name=first_name,
+        last_name=last_name
+    )
+    return creating_user
+
+
+def get_user(user_id: int) -> AUTH_USER_MODEL:
+    return get_user_model().objects.get(id=user_id)
+
+
+def update_user(user_id: int,
+                username: Optional[str] = None,
+                password: Optional[str] = None,
+                email: Optional[str] = None,
+                first_name: Optional[str] = None,
+                last_name: Optional[str] = None) -> QuerySet:
+    get_user_id = get_user_model().objects.get(id=user_id)
```

Comment:
get_user_id so strange name
Rename it to user

Reply:
Not fixed